guides/common/modules/proc_installing-the-sos-package.adoc

```diff
@@ -1,14 +1,16 @@
 [id="installing-the-sos-package_{context}"]
-= Installing the SOS Package on the Base Operating System
+= Installing the sos package on the base operating system
+
+Install the `sos` package on the base operating system so that you can collect configuration and diagnostic information from a {EL} system.
 
```

Comment:
```suggestion
```

AFAIK this would result in two paragraphs for Satellite docs; the first one consisting of only one sentence.

Reply:
fixed